MEDO v0.5.owl

```diff
@@ -804,6 +975,536 @@ LCME Glossary of Terms for LCME Accreditation Standards and Elements 2015-2016,
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#formal_summative_research_or_project_assessment">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo//MEDO_0000105"/>
+        <obo:IAO_0000115>A type of formal summative assessment of activities and outcomes (e.g., posters, presentations, reports, etc.) of a project in which the learner participated or conducted research.</obo:IAO_0000115>
+        <obo:IAO_0000119>MedBiquitous - AM16: Assessment of activities and outcomes (e.g., posters, presentations, reports, etc.) of a project in which the learner participated or conducted research (Dyrbye, Davidson, &amp; Cook, 2008)
+
+Synonymous with/Includes: Multi-Media Production</obo:IAO_0000119>
+        <rdfs:label>formal summative research or project assessment</rdfs:label>
+        <skos1:related>MedBiquitous - AM16: Research or Project Assessment</skos1:related>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/medo.owl#formal_summative_self-assessment -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#formal_summative_self-assessment">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo//MEDO_0000105"/>
+        <obo:IAO_0000115>A type of formal summative assessment where student observes, evaluates, and reflects upon one’s own deficiencies, achievements, behavior or professional performance, and competencies. The assessment is completed by the learner to reflect upon and critically assess their own performance against a set of established criteria.</obo:IAO_0000115>
+        <obo:IAO_0000119>MedBiquitous - AM17: The process of evaluating one’s own deficiencies, achievements, behavior or professional performance and competencies (Institute for International Medical Education, 2002); Assessment completed by the learner to reflect and critically assess his/her own performance against a set of established criteria (Gordon, 1991) (NOTE: Does not refer to NBME Self-Assessment)
+
+Synonymous with/Includes: Observation of self on video
+
+Does not include: NBME Self-Assessment (National Board of Medical Examiners (NBME) Self-Assessment Services, n.d.)</obo:IAO_0000119>
+        <rdfs:label>formal summative self-assessment</rdfs:label>
+        <skos1:related>MedBiquitous - AM17: Self-Assessment</skos1:related>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/medo.owl#formal_summative_teacher_observation_and_feedback -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#formal_summative_teacher_observation_and_feedback">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo//MEDO_0000105"/>
+        <obo:IAO_0000115>A type of formal summative assessment where teacher or observer(s) observes, assesses, and provides written or oral subjective feedback of a learner&apos;s work or performance.</obo:IAO_0000115>
```

Comment:
As above, the Medbiquitous definition specifies only written feedback, not oral. Is it your intention to have a broader definition? If so, should there be two child classes, one for 'written formal formative teacher observation and feedback' and a second for 'oral formal formative teacher observation and feedback'? The Medbiquitous AM10 could then be linked to the written one. Also, I would write "A type of formal summative assessment in which one or more teachers or observers observe, assess, and provide written or oral subjective feedback of a learner's work or performance.

Furthermore, is it your intention that AM10 maps to both the 'formal summative teacher observation and feedback' and ?the 'formal formative teacher observation and feedback' classes? (the answer can be yes--just want to make sure)